update docs for MC-1162 (#150)
hazelcast/management-center-docs: docs/modules/clusters/pages/clients.adoc

You cannot view any more details about this client other than what is displayed on the *Clients* page.

> NOTE: MC-Clients are *not included* in Client Counts on xref:getting-started:user-interface.adoc#menu[the Menu on the left-hand side] and xref:deploy-manage:connecting-to-clusters-ui.adoc[Cluster Connections Screen]

Reviewer: ```suggestion
NOTE: The MC-Client is *not included* in the total client count that is displayed in the xref:getting-started:user-interface.adoc#menu[left-hand menu] and the xref:deploy-manage:connecting-to-clusters-ui.adoc[Cluster Connections page]
```
